scripts/release_support/changelog.sh

```diff
@@ -58,8 +58,38 @@ export TZ=$time_zone;ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ
 
 pushd $clone_dir/clone
 
-wget -q $BUILD_LOCATION/$START_BUILD/dev/RELEASE.INFO -O start_build_manifest.txt
-wget -q $BUILD_LOCATION/$TARGET_BUILD/dev/RELEASE.INFO -O target_build_manifest.txt
+if [ -z $BUILD_LOCATION ]; then
+	NOT_DOCKER_IMAGE=$(echo "$START_BUILD"|grep RELEASE.INFO|wc -l)
+	if [ $NOT_DOCKER_IMAGE == "0" ]; then
+		docker pull "$START_BUILD"
+		firstfilecontainerid="$(docker run -it -d $START_BUILD bash)"
+		docker cp "$firstfilecontainerid":RELEASE.INFO start_build_manifest.txt
+		START_BUILD=$(grep "BUILD:" start_build_manifest.txt| awk '{print $2}'|tr -d '"')
+		docker rm -f "$firstfilecontainerid"
```

Comment:
You can combine all above in single command as below - 
`docker run --rm ghcr.io/seagate/cortx-all:2.0.0-563-PI53.2  cat /opt/seagate/cortx/RELEASE.INFO > start_build_manifest.txt`

Reply:
Thanks for simple command. Combine above commands in single one.